docs/modules/links.md

```diff
@@ -0,0 +1,125 @@
+# Dynamic Links
+
+[Firebase Dynamic Links](https://firebase.google.com/docs/dynamic-links/) allows you to create and receive links on both Android and iOS platforms. Assuming the installation instructions have been followed, Firebase Dynamic Links is ready to go.
+
+
+All Dynamic Links operations are accessed via `firebase.links()`
+
+## Create Dynamic Links
+
+RNFirebase mimics [The REST API](https://firebase.google.com/docs/dynamic-links/rest) for Dynamic Links creation.
+The differences from the REST API are:
+1. The input for the methods is a javascript object instead of a JSON object.
+2. The response contains the URL string only.
+
+### Methods
+
+#### `createDynamicLink(parameters: Object): Promise<String>`
+
+Creates a long dynamic link.
+
+```javascript
+firebase.links().createDynamicLink({
+  dynamicLinkInfo: {
+    dynamicLinkDomain: "abc123.app.goo.gl",
+    link: "https://example.com/",
+    androidInfo: {
+      androidPackageName: "com.example.android"
+    },
+    iosInfo: {
+      iosBundleId: "com.example.ios"
+    }
+  }
+}).
+then((url) => {
+  // ...
+});
+```
+
+#### `createShortDynamicLink(parameters: Object): Promise<String>`
+
+Creates a short dynamic link.
+
+```javascript
+firebase.links().createShortDynamicLink(dynamicLinkInfo: {
+    dynamicLinkDomain: "abc123.app.goo.gl",
+    link: "https://example.com/",
+    androidInfo: {
+      androidPackageName: "com.example.android"
+    },
+    iosInfo: {
+      iosBundleId: "com.example.ios"
+    }
+  }
+}).
+then((url) => {
+  // ...
+});
+```
+### Parameters
+
+Only the following parameters are currently supported:
+
+```javascript
+{
+  dynamicLinkInfo: {
+    dynamicLinkDomain: string,
+    link: string,
+    androidInfo: {
+      androidPackageName: string,
+      androidFallbackLink: string,
+      androidMinPackageVersionCode: string,
+      androidLink: string
+    },
+    iosInfo: {
+      iosBundleId: string,
+      iosFallbackLink: string,
+      iosCustomScheme: string,
+      iosIpadFallbackLink: string,
+      iosIpadBundleId: string,
+      iosAppStoreId: string
+    },
+    socialMetaTagInfo: {
+      socialTitle: string,
+      socialDescription: string,
+      socialImageLink: string
+    }
+  },
+  suffix: {
+    option: "SHORT" or "UNGUESSABLE"
+  }
+}
+```
+?> Please note that when using `androidInfo` or `iosInfo`, `androidPackageName` and `iosBundleId` are mandatory (respectively).
```

Comment:
`?> ` prefix here is on purpose?

Reply:
Yes. see: https://github.com/rmrs/react-native-firebase/blob/omer_links/docs/modules/firestore.md vs. https://invertase.io/react-native-firebase/#/modules/firestore